### PR TITLE
Update the option for docker build

### PR DESCRIPTION
**Pull request by yuexiao-wang**
**- What I did**
1. Update the option 'network' for docker build
2. Update the format of squash  for docker build
3. Update the man document 

**- How I did it**

**- How to verify it**

**- Description for the changelog**
Update for keep consistency for docker build

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

File: cli/command/image/build.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func NewBuildCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.StringSliceVar(&options.cacheFrom, "cache-from", []string{}, "Images to consider as cache sources")
 	flags.BoolVar(&options.compress, "compress", false, "Compress the build context using gzip")
 	flags.StringSliceVar(&options.securityOpt, "security-opt", []string{}, "Security options")
-	flags.StringVar(&options.networkMode, "network", "default", "Connect a container to a network")
+	flags.StringVar(&options.networkMode, "network", "default", "Set the networking mode for the run commands during build")
```
Comment (AkihiroSuda) [inline cli/command/image/build.go]: SGTM, but I think `RUN instructions` should be better than `run commands`
https://docs.docker.com/engine/reference/builder/

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/build.md [hunk 2]
```diff
@@ -54,7 +53,7 @@ Options:
                                 The format is `<number><unit>`. `number` must be greater than `0`.
                                 Unit is optional and can be `b` (bytes), `k` (kilobytes), `m` (megabytes),
                                 or `g` (gigabytes). If you omit the unit, the system uses bytes.
-  --squash                      Squash newly built layers into a single new layer (**Experimental Only**) 
+  --squash                      Squash newly built layers into a single new layer (**Experimental Only**)
```
Comment (AkihiroSuda) [inline docs/reference/commandline/build.md]: You can fix this indentation as well

**Comment by yuexiao-wang** [comment]
@AkihiroSuda PTAL
Done!

File: docs/reference/commandline/build.md (hunk 2 above)
Comment (yuexiao-wang) [inline docs/reference/commandline/build.md]: @AkihiroSuda PTAL
Done.
Sorry, response to it late.

**Review by thaJeztah — APPROVED** [review]
LGTM

File: man/docker-build.1.md [hunk 3]
```diff
@@ -130,8 +130,9 @@ set as the **URL**, the repository is cloned locally and then sent as the contex
 unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
 
 **--network**=*NETWORK*
-  
-
+Sets the networking mode for the run commands during build.
+Supported standard values are: `bridge`, `host`, `none` and `container:<name|id>`.
+Any other value is taken as a custom network's name or ID which this container should connect to.
```
Comment (AkihiroSuda) [inline man/docker-build.1.md]: Please add an empty line here

Comment (yuexiao-wang) [inline man/docker-build.1.md]: @AkihiroSuda PTAL
Done and thanks

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Pull request merged**